tests/ui_tests/report.py

```diff
@@ -0,0 +1,104 @@
+import base64
+import filecmp
+from distutils.dir_util import copy_tree
+from itertools import zip_longest
+
+import dominate
+from dominate.tags import div, h1, hr, i, img, p, table, td, th, tr
+
+from . import download
+
+
+def _image(src):
+    with td():
+        if src:
+            # open image file
+            image = open(src, "rb")
```

Comment:
tiny nitpick but still: `with open(src, "rb") as f: ...` makes sure the image is closed afterwards

Reply:
also as it turns out `src` is a Path, so instead `image = src.read_bytes()` :)